src/fsc/fsc.targets

```diff
@@ -44,6 +44,26 @@
     <ProjectReference Include="$(MSBuildThisFileDirectory)../FSharp.DependencyManager.Nuget/FSharp.DependencyManager.Nuget.fsproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This Item group should probably look a bit more like this:
````
  <ItemGroup>
    <MIBCPackage Include="optimization.windows_nt-x86.mibc.runtime"   Version="$(optimizationwindows_ntx86MIBCRuntimeVersion)" />
    <MIBCPackage Include="optimization.windows_nt-x64.mibc.runtime"   Version="$(optimizationwindows_ntx64MIBCRuntimeVersion)" />
    <MIBCPackage Include="optimization.windows_nt-arm64.mibc.runtime" Version="$(optimizationwindows_ntarm64MIBCRuntimeVersion)" />
    <MIBCPackage Include="optimization.linux-x64.mibc.runtime"        Version="$(optimizationlinuxx64MIBCRuntimeVersion)" />
    <MIBCPackage Include="optimization.linux-arm64.mibc.runtime"      Version="$(optimizationlinuxarm64MIBCRuntimeVersion)" />

    <MIBCPackage Include="optimization.linux-x64.mibc.runtime"        Version="$(optimizationlinuxx64MIBCRuntimeVersion)" />
  </ItemGroup>
````

There may be some issue with missing packages when doing build from source.  We can work out the correct approach in that build scenario, after we see it fail.  Even then it's likely to be just exclude the -nt- packages.